examples/with-federated-passkeys/src/pages/index.tsx

```diff
@@ -112,6 +113,11 @@ export default function Home() {
     // https://www.w3.org/TR/webauthn-2/#sctn-sample-registration
     const attestation = await getWebAuthnAttestation({
       publicKey: {
+         authenticatorSelection: {
+          residentKey: "preferred",
+          requireResidentKey: false,
+          userVerification: "preferred"
```

Comment:
hey @octavien-mural sorry for our linter being so pushy, but we may need another trailing comma here :) 

to be safe, you can also try running `pnpm -w prettier-all:write`, in case anything else is missed (doesn't look like it though)

Reply:
Apologies, just pushed again! I believe I have different prettier settings so I'm unable to run the command with the correct styling